script/run-remote-dev-match

```diff
@@ -0,0 +1,162 @@
+    ENDC = '\033[0m'
+    return print(f"{BOLD}{FAIL}{message}{ENDC}")  # noqa:T001
+
+
+def print_fatal(message: str) -> None:
+    return print_error(message, strong=True)
+
+
+def run_match() -> str:
+    try:
+        return subprocess.check_output([
+            'webots',
+            '--batch',
+            '--stdout',
+            '--stderr',
+            '--minimize',
+            '--mode=realtime',
+            str(REPO_ROOT / 'worlds' / 'Arena.wbt'),
+        ], stderr=subprocess.STDOUT, universal_newlines=True)
+    except FileNotFoundError:
+        print_fatal(
+            "Could not find webots. Check that you have it installed and on your PATH",
+        )
+        exit(1)
+    except subprocess.CalledProcessError as e:
+        try:
+            log_text = controller_utils.get_competition_supervisor_log_filepath().read_text()
+        except FileNotFoundError:
+            print_fatal(
+                f"Simulation errored (exit code {e.returncode}). "
+                "No supervisor logs were found - Webots may have crashed.",
+            )
+        else:
+            # There are potentially a large number of lines in the logs and any
+            # errors are likely to be at the end of the logs. Additionally, the
+            # user's focus is initially likely to be at the end of the output.
+            # To ensure that failures are clear we therefore put the message that
+            # there was a failure last in the output (and in bold).
+            print_error(log_text)
+            print_fatal(
+                f"Simulation errored (exit code {e.returncode}). "
+                f"Competition supervisor logs are above.",
+            )
+
+        exit(1)
+
+
+def archive_match_outputs(archives_dir: Path, match_logs: str) -> None:
+    """
+    Copy the recordings and logs to the output directory.
+    """
+
+    output_dir = archives_dir / "output" / datetime.now().strftime('%c')
+    output_dir.mkdir(exist_ok=True, parents=True)
+
+    (output_dir / "logs.txt").write_text(match_logs)
+
+    recording_dir = controller_utils.get_recording_stem().parent
+
+    # We don't want to rely on the recordings going into a directory which only
+    # contains recording data, so instead we copy explicitly the things which we
+    # know will have been output.
+
+    for path in recording_dir.glob('*'):
+        shutil.copy(path, output_dir)
+
+    textures_dir = recording_dir / 'textures'
+
+    # If we don't have textures, the directory won't exist.
+    if textures_dir.is_dir():
+        shutil.copytree(textures_dir, output_dir / 'textures')
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'source_dir',
+        help=(
+            "The directory containing the teams' robot code. This directory "
+            "will also be used as the root for storing the resulting logs and "
+            "recordings."
+        ),
+        type=Path,
+    )
```

Comment:
Given that the argument is also used as a destination, possibly "source dir" is slightly misleading?